src/harness/testcases/WINNF_FT_S_WDB_testcase.py

```diff
@@ -170,38 +276,42 @@ def test_WINNF_FT_S_WDB_2(self, config_filename):
     # Load the configuration file
     config = loadConfig(config_filename)
 
+    # Very light checking of the config file.
+    self.assertNotEqual(config['registrationRequests'], None)
+
+    registration_requests = {
+        'registrationRequest': config['registrationRequests']
+    }
+
     # Step 1: Register device B and ensure failure
-    request = {'registrationRequest': config['registrationRequest']}
-    response = self._sas.Registration(request)['registrationResponse']
+    # Inject FCC ID and User Id of the device(s) into UUT.
+    for registration_request in config['registrationRequests']:
+      self._sas_admin.InjectFccId({'fccId': registration_request['fccId']})
+      self._sas_admin.InjectUserId({'userId': registration_request['userId']})
 
-    # Check the register response code is 103 (INVALID_VALUE).
-    self.assertEqual(response['response']['responseCode'], 103)
+    # Send registration request for CAT 'B' CBSD 'C' to SAS UUT.
```

Comment:
This comment is misleading since it is sending multiple registration requests at once. Consider just labeling as "Send registration requests."

Reply:
Updated   the statement as per the comment